bench/perf_run_py/perf_stat_collection.py

```diff
@@ -119,5 +119,63 @@ def all_stats():
             PerfStatMin(short_label="min_read_throughput",
                         pattern=r'updates,',
                         input_offset=4,
-                        output_label='Min read throughput')
+                        output_label='Min read throughput'),
+            PerfStatCount(short_label="warning_operations",
```

Comment:
In the [doc](https://docs.google.com/document/d/1ZZx9cmM0M_zgeT56EswJA-TgrIE6RrxX3aw_86PyZ4U/edit#heading=h.u9klwjln1krx) under the `Job: wiredtiger-many-dhandles` you'll see that all of the other stats are also gathered (not plotted). Hence, I thought if someone wanted to plot them in the future they wouldn't have to go with the trouble of including them.

Reply:
ok, so that will mean when we shut down Jenkins we no longer track these stats..
I'm leaning towards adding them all into evergreen so that we can maintain historical values for them. Let's discuss with @jeremythorp

Reply:
I'd suggest adding all the stats (and plotting them) - people can ignore them if they are not relevant (or even remove them later if they are getting in the way).

Reply:
work done in [5cc5a18](https://github.com/wiredtiger/wiredtiger/pull/7223/commits/5cc5a18798532d26dd85751c8b449052a492e539)